src/test-helpers.tsx

```diff
@@ -195,7 +195,7 @@ export const createFakeHistory = ({
 
 export const createContextWithFakeRouter = ({
   history = createFakeHistory(),
-  location = null,
+  location = null as Location<{}> | null,
```

Comment:
Huh. This is hurting my brain.

Reply:
It forces `location` to be a nullable `Location` object.